### PR TITLE
Xcode 8.3 migration (and crash fix)

### PR DESCRIPTION
**Pull request by nap-sam-dean**

### CODE REVIEW

File: Pod/Core/ClassHelperMethods.swift [hunk 1]
```diff
@@ -20,24 +20,37 @@ import Foundation
...
-            }
+
+    for i in 0..<actualClassCount {
+
+        guard let currentClass = allClasses[Int(i)] else {
+            continue
+        }
+
+        guard class_getSuperclass(currentClass) != nil else {
+            print("\(currentClass) has no superclass")
```
Comment (smaljaar) [inline Pod/Core/ClassHelperMethods.swift]: I think you can remove the print statement, I don't really like seeing the list of classes without super class on every run in the log 🙂

File: Pod/Core/ClassHelperMethods.swift [hunk 2]
```diff
@@ -20,24 +20,37 @@ import Foundation
...
+            continue
+        }
+
+        guard !blackList.contains(Bundle(for: currentClass)) else {
+            continue
+        }
+
+        if currentClass is T {
+            matches.append(currentClass as! T)
+            print("Found \(currentClass)")
```
Comment (smaljaar) [inline Pod/Core/ClassHelperMethods.swift]: I think you can remove the print statement, I don't really like seeing the list of classes with a super class on every run in the log 🙂

**Comment by smaljaar** [comment]
I verified this fix and the crash is gone. 👍

File: Pod/Core/ClassHelperMethods.swift (hunk 2 above)
Comment (nap-sam-dean) [inline Pod/Core/ClassHelperMethods.swift]: Blast, they're no meant to still be in there!

**Pull request merged**